internal/distro/distro_test_common/distro_test_common.go

```diff
@@ -231,32 +245,9 @@ func GetTestingPackageSpecSets(packageName, arch string, pkgSetNames []string) m
 // defined by the provided ImageType, which is useful for unit testing.
 func GetTestingImagePackageSpecSets(packageName string, i distro.ImageType) map[string][]rpmmd.PackageSpec {
 	arch := i.Arch().Name()
-	imagePackageSets := make([]string, 0, len(i.PackageSets(blueprint.Blueprint{})))
-	for pkgSetName := range i.PackageSets(blueprint.Blueprint{}) {
+	imagePackageSets := make([]string, 0, len(i.PackageSets(blueprint.Blueprint{}, nil)))
+	for pkgSetName := range i.PackageSets(blueprint.Blueprint{}, nil) {
 		imagePackageSets = append(imagePackageSets, pkgSetName)
 	}
 	return GetTestingPackageSpecSets(packageName, arch, imagePackageSets)
 }
-
-// Ensure that all package sets defined in the package set chains are defined for the image type
```

Comment:
I added comment to the `MakePackageSetChains()` function. I think that it could be adjusted to produce error in the case that was previously tested by this unit test. If done, then this unit test could be replaced by simply calling `PackageSets()` on each image type with `nil` repos.